### DISCUSSION TITLE
Color my <body> tag

### DISCUSSION THREAD

**Discussion by hornta** [tailwindlabs/tailwindcss#3079] [Help] [answered]
I want to set a color on my <body> tag but it doesn't work when I set the bg-dark-900 class on the body tag since it's in the index.html (I'm using CRA). Can I some how grab a certain color from Tailwind using CSS-variables so that I can write in in my index.css? 

I tried background-color: var(--color-grey-900) but without any luck :(

**Comment by chr-ge** [comment, marked as answer]
If you follow the [Tailwindcss CRA guide](https://tailwindcss.com/docs/guides/create-react-app), you should be able to use tailwind classes in the `index.html` file with no issues.
```html
<body class="bg-grey-900">
   ...
</body>
```

**Reply by hornta** [reply]
Nice! I must've missed that. But if I in the future should need to use Tailwind variables in my css-files can I do it somehow? I'm about to switch to snowpack and I saw they had a Tailwind section but no mention of using classes outside of the build pipiline.

**Reply by chr-ge** [reply]
I believe to use Tailwind variables you would need to use a package like [tailwind-css-variables](https://github.com/omarkhatibco/tailwind-css-variables). 
You can also take a look at [using-with-preprocessors](https://tailwindcss.com/docs/using-with-preprocessors#variables).
